src/selectors/markdownSelectors.js

```diff
@@ -0,0 +1,3 @@
+export const getAllMarkdowns = state => {
```

Comment:
there is only one markdown file. should probably be `getMarkdown`

Reply:
Agree on all accounts. We changed the PR to merge to dev. And fixed the other pieces!